### DISCUSSION TITLE
Welcome to Thanox Discussions!

### DISCUSSION THREAD

**Discussion by Tornaco** [Tornaco/Thanox#321] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by zhouzwow** [comment]
为什么我只要一安装这个APP，微信小程序就打开非常慢，要转半天的圈，一删除APP微信小程序就秒打开。
